python/cudf/cudf/tests/test_multiindex.py

```diff
@@ -2162,3 +2162,14 @@ def test_multi_index_contains_hashable():
         lfunc_args_and_kwargs=((),),
         rfunc_args_and_kwargs=((),),
     )
+
+
+@pytest.mark.parametrize("array", [[1, 2], [1, None], [None] * 2])
```

Comment:
Slightly less cognitive overhead.
```suggestion
@pytest.mark.parametrize("array", [[1, 2], [1, None], [None, None]])
```